corehq/apps/app_manager/app_translations/app_translations.py

```diff
@@ -783,6 +795,22 @@ def _update_case_list_translations(sheet, rows, app):
         elif row['case_property'] == 'case_list_form_label':
             case_list_form_label = row
 
+        # If it's a tab header, don't add it to condensed rows
+        elif re.search(r'^Tab \d+$', row['case_property']):
```

Comment:
@orangejenny so just confirming that this
checks for text "Tab some_integer" in the case_property field.. and if it is, it goes further and adds it in `detail_tab_headers`. 
If yes,
Is this text "Tab" in the field set internally and would always be there?

What does `condensed` rows here refer to?

Reply:
Yep, this checks for "Tab " plus an integer, and that text does get set by the code that generates the bulk translation download (in the changes above, around line 300).

`condensed_rows` is a list containing all of the provided translations, with one element for each case property, and ultimately it's the data used to actually update the app. It's condensed compared to the list `rows` provided, which comes directly from the uploaded file and may contain multiple rows for the same case property (case properties ID mappings, graphs, etc. have multiple translate-able strings and a row for each of those strings).